### ISSUE TITLE
Improve `version` default in Metadata Hash

### ISSUE THREAD

**Issue by acuarica** [acuarica/evm#3]
When there is no `version` detected within the Metadata Hash, it defaults to `0.5.5`. Investigate whether this is accurate, and if not, use a better default.

**Comment by acuarica** [comment]
Closed in c0ed39592b947d3332f37077ca876159cfc18fed.

**Comment by acuarica** [comment]
Metadata stats for Etherscan tests has been added in 6a3c2d7.

```
$ yarn test:smoke -g etherscan
[...]
  Metadata Stats
    • Contracts with no metadata 37
    • Protocols ipfs|bzzr
    • SOLC Versions 0.6.12|0.8.19|0.8.18|0.8.10|0.8.14|0.8.7|0.8.0|0.8.13|0.8.17|0.8.16|0.8.9|0.8.15|0.8.4|0.8.2|0.5.16|0.5.17|0.6.6|0.7.6|0.8.5|0.8.11|0.8.12|0.8.6|<0.5.9|0.5.15|0.7.1|0.8.1|0.7.5|0.7.0|0.5.12|0.8.
8|0.6.10|0.6.0|0.6.11|0.8.3|0.6.8|0.6.4|0.6.7|0.6.2|0.7.4
```
